enhancements/authentication/custom-scc-preemption-prevention.md

```diff
@@ -0,0 +1,249 @@
+---
+title: custom-scc-preemption-prevention
+authors:
+  - "@s-urbaniak"
+reviewers:
+  - "@stlaz"
+  - "@ibihim"
+approvers:
+  - "@deads2k"
+api-approvers:
+  - "None"
+creation-date: 2023-05-03
+last-updated: 2023-05-03
+tracking-link:
+  - https://issues.redhat.com/browse/AUTH-132
+see-also:
+  - "/enhancements/this-other-neat-thing.md"
+replaces:
+  - "/enhancements/that-less-than-great-idea.md"
+superseded-by:
+  - "/enhancements/our-past-effort.md"
+---
+
+# Preventing preemption of system SCCs by custom SCCs for core OpenShift components
+
+## Summary
```

Comment:
Nit: I would mention the preference for the hard protection in the summary as well.

Reply:
good suggestion, fixed.